### DISCUSSION TITLE
404 File not found  The site configured at this address does not contain the requested file.  If this is your site, make sure that the filename case matches the URL as well as any file permissions. For root URLs (like http://example.com/) you must provide an index.html file.  Read the full documentation for more information about using GitHub Pages.

### DISCUSSION THREAD

**Discussion by khangdang12324** [community/community#138510] [Other Feature Feedback, Questions, & Ideas]
404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages.
https://github.com/khangdang12324/KhangResume.github.io
https://github.com/khangdang12324/KhangResume.github.io

**Comment by queenofcorgis** [comment]
Thanks for your post, @khangdang12324 ! We worked with the Pages team to create a [guide](https://github.com/orgs/community/discussions/61073) to help you troubleshoot this error. Check out the [Pages 404 error guide](https://github.com/orgs/community/discussions/61073) for assistance. And don't forget to spread the knowledge 💡 and the guide to your fellow community members. 

Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by davevad93** [comment]
Hi @khangdang12324 ,

The issue with your [My CV](https://github.com/khangdang12324/KhangResume.github.io) repo is that the structure isn't correct, `index.html` isn't located in the root. This is your structure: 

`/My CV`
`└── /CV`
           `      ├── /Image`
           `      ├── cv.pdf`
           `      ├── index.html`
           `      └── style.css`


But the right structure is as follows:

`/My CV`
                                                     `├── /Image`
`├── cv.pdf`
`├── index.html`
`└── style.css`

So basically delete the `CV` folder and put its contents in the root of the repo.

**Comment by Miguel1995c** [comment]
Hello, @queenofcorgis , the question is inappropriate but. Do you know how long it takes for GitHub support to respond?
 my main GitHub account was suspended I already submitted support ticket N#: 2994789 and N#: 2994844.

**Comment by leonlime** [comment]
Hello @khangdang12324 , i think you need to use same username to create your website.

like 
https://github.com/khangdang12324/khangdang12324.github.io

I think is mandatory to use the same name as your username.
